src/tests/libecalc/presentation/yaml/yaml_models/test_yaml_model_errors.py

```diff
@@ -0,0 +1,39 @@
+from io import StringIO
+
+import pytest
+from libecalc.presentation.yaml.yaml_entities import ResourceStream
+from libecalc.presentation.yaml.yaml_models.exceptions import YamlError
+from libecalc.presentation.yaml.yaml_models.yaml_model import YamlModel, YamlModelType
+
+invalid_models = [
+    (
+        """
+`INSTALLATIONS: []
+""",
+        "invalid_token_start",
```

Comment:
should this message be presented to users? Should be consider making it more understandable? (if so)

Reply:
That's not the expected value in the test, it is just a description/name of the model. This test only makes sure the error raised is YamlError. I haven't done much with the error messages, so we will show what pyyaml/ruamel reports. That may not be understandable to users.